src/tests/ftest/control/dmg_telemetry_io_basic.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from avocado.core.exceptions import TestFail
+
+from ior_test_base import IorTestBase
+from general_utils import get_remote_file_size, run_pcmd
+from telemetry_test_base import TestWithTelemetry
+from telemetry_utils import TelemetryUtils
+
+
+class TestWithTelemetryIOBasic(IorTestBase,TestWithTelemetry):
+    # pylint: disable=too-many-ancestors
+    """Test telemetry engine io basic metrics.
+
+    :avocado: recursive
+    """
+
+    def verify_io_test_metrics(self, io_test_metrics, metrics_data):
+        """ Verify telemetry io metrics from metrics_data.
+
+        Args:
+            io_test_metrics (list): list of telemetry io metrics.
+            metrics_data (dict): a dictionary of host keys linked to a
+                                 list of io metric names.
+        """
+        for host in self.hostlist_servers:
+            self.log.info("==Host: %s", host)
+            for name in io_test_metrics:
+                self.log.info("  --telemetry metric: %s", name)
+                for i in range(len(metrics_data)):
```

Comment:
(pylint-consider-using-enumerate) Consider using enumerate instead of iterating with range and len